.circleci/config.yml

```diff
@@ -14,6 +14,9 @@ workflows:
       - "test-arangodb-3_6":
           requires:
             - "test-arangodb-3_5"
+      - "test-arangodb-3_7":
+          requires:
+            - "test-arangodb-3_6"
```

Comment:
I think we should require `build` instead. It could potentially let the tests for each versions run in parallel.